inf-clojure.el

```diff
@@ -343,6 +343,12 @@ number (e.g. (\"localhost\" . 5555))."
    (stringp (car x))
    (numberp (cdr x))))
 
+(defcustom inf-clojure-custom-startup
+   nil
+   "Form to be used to start inf-clojure.
```

Comment:
That's a command, not a form, so I'd reflect it in the name of the defcustom.

Reply:
even if its a cons cell of host and port? Custom startup sounds vague enough that it encompasses both. If it had command I might think it was a string to start up rather than allowing for the connection tuple. I don't have strong opinions though so happy to take a final answer and commit it.